docs/migrating-from-mdl.md

```diff
@@ -0,0 +1,318 @@
+# Migrating from Material Design Lite
+
+Material Components for the web (MDC-Web) is the successor to the Material Design Lite (MDL) project.
+
+While the philosophy behind the two projects is quite similar, migrating to MDC-Web requires a number of changes, from
+class names to different DOM structures. In addition, there are several choices to be made regarding component
+initialization, how to depend on MDC-Web, and theming/styling mechanisms.
+
+This document attempts to summarize and guide you through the work involved. Let’s get started!
+
+> **Note:** If you’re thinking of migrating your application to MDC-Web, please bear in mind that it’s still in an alpha
+state and thus not recommended for production.
+
+
+## Depending on MDC-Web
+
+MDL is distributed on NPM, Bower, and through its own CDN. MDC-Web is currently only available on NPM.
+
+While MDL is an all-in-one solution, MDC-Web has been designed to be modular, allowing you to make the choice of either
+pulling in everything, or just the packages you want.
+
+### NPM
+
+MDC-Web is made available on [NPM](https://www.npmjs.com/), with packages living under the `@material` namespace.
+
+In order to install e.g. the button component, you can run:
+
+```
+npm install --save @material/button
+```
+
+Some packages serve as dependencies for others, so don’t be surprised if you end up with multiple packages in your
+`node_modules` folder! There are no runtime dependencies outside of MDC-Web, so the dependency tree should remain small.
+
+If you want all of MDC-Web, you can pull in the meta package:
+
+```
+npm install --save material-components-web
+```
+
+### Content Distribution Network (CDN)
+
+While MDL is available over a CDN, there’s currently no equivalent in MDC-Web (it’s in the plans for the future,
+though!).
+
+In the meantime, you can take advantage of the [unpkg CDN](https://unpkg.com/), which automatically provides
+distribution for all NPM packages.
+
+For easily getting all MDC-Web CSS:
+
+```
+https://unpkg.com/material-components-web@latest/dist/material-components-web.css
+```
+
+And JS:
+
+```
+https://unpkg.com/material-components-web@latest/dist/material-components-web.js
+```
+
+Check the [unpkg CDN main page](https://unpkg.com/) for more information on how to request particular version ranges.
+
+There’s also the option of getting individual packages, rather than the `material-components-web` meta-package. However,
+since unpkg only serves up package contents without resolving dependencies, you’d need to maintain the dependency tree
+yourself, which is not recommended.
+
+
+## Initializing components
```

Comment:
That's the same as in MDL, but I'll add a bit about how MDC-Web doesn't write any DOM for you, unlike MDL.